homeassistant/components/tplink/diagnostics.py

```diff
@@ -42,10 +40,10 @@
 
 
 async def async_get_config_entry_diagnostics(
-    hass: HomeAssistant, entry: ConfigEntry
+    hass: HomeAssistant, entry: TPLinkConfigEntry
 ) -> dict[str, Any]:
     """Return diagnostics for a config entry."""
-    data: TPLinkData = hass.data[DOMAIN][entry.entry_id]
+    data = entry.runtime_data
```

Comment:
Minor: some places call this `entry` and others call it `config_entry`.  I prefer the shorter but we should pick one or the other and it doesn't matter so much as long as we are consistent

Reply:
Done in https://github.com/home-assistant/core/pull/120025